pypuppetdb/types.py

```diff
@@ -128,22 +176,36 @@ def __str__(self):
     def __unicode__(self):
         return self.__string
 
+    def events(self):
+        """Get all events for this report."""
+        return self.__api.events(query=self.__query_scope)
```

Comment:
Right. I've already created a PR for puppetboard based on this PR + this patch. I'd love it if this could be published soon, as I have more patches based on the v4 API waiting, too :-)